ergoCubSN001/wrappers/inertials/alljoints-inertials_remapper.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE devices PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+
+    <device xmlns:xi="http://www.w3.org/2001/XInclude" name="alljoints-inertials_remapper" type="multipleanalogsensorsremapper">
+        <param name="OrientationSensorsNames">
+            (head_imu_0 l_arm_ft l_leg_ft l_foot_ft r_arm_ft r_leg_ft r_foot_ft)
```

Comment:
It seems that there are a few sensors missing:
* `waist_imu_0`

In the foot there are two FT sensors, are you sure that it exists a sensor called `r_foot_ft`?

Reply:
Done https://github.com/robotology/robots-configuration/pull/632/commits/354770cce42819237ab46dec98d02f5dab82007a